vnext/ReactUWP/Views/TouchEventHandler.h

```diff
@@ -5,14 +5,15 @@
 #include <folly/dynamic.h>
 #include <optional>
 #include <set>
-
+#include <UI.Xaml.Documents.h>
 #include <IReactInstance.h>
 #include "XamlView.h"
 
 namespace winrt {
 using namespace Windows::UI;
 using namespace xaml;
 using namespace xaml::Controls;
+using namespace xaml::Documents;
```

Comment:
we should try to minimize the number of things in the winrt namespace

Reply:
I remember we wanted to use namespace aliasing - should I remove this line and use the full namespace name everywhere I refer to ..Documents..?
(Why do want to limit what we put here?)

Reply:
@asklar can you elaborate? Why don't we want this, and if so should use the fully-qualified name for this namespace? :)